### DISCUSSION TITLE
"Error when trying to authenticate with LPAD on windows server."

### DISCUSSION THREAD

**Discussion by Willycrf** [requarks/wiki#6037] [Error / Bug Report]
Hello everyone!!

"I have the following problem in my setup, when I try to perform LDAP authentication, I can't get it to work, I get the following error, **"Invalid email / username or password",** I have everything set up on a Windows Server. I can't see the logs, since there is no folder that references them and I do have the flag that references them activated."

> OS: Windows Server 2019
> Wiki.js version: 2.5.295
> Database engine: PostgreSQL 15.1-1 x64
> Node version: 16.19.0 x64

![image](https://user-images.githubusercontent.com/122536734/212074365-086fec05-dc7a-4a6c-9b42-87fe69e41dfb.png)
![image](https://user-images.githubusercontent.com/122536734/212074447-2fd5b344-5108-427d-97ec-591349d106a8.png)

**Comment by fnafox** [comment]
Try entering just the `sAMAccountName` of the binding account into the `Admin Bind DN` text box instead of LDIF

**Comment by Willycrf** [comment]
"I have tried what has been mentioned but it still does not work, is there a way to see the error in logs?"

![image](https://user-images.githubusercontent.com/122536734/212299482-e0b72310-e3d4-408d-8086-a501dcbbc6ec.png)

**Reply by fnafox** [reply]
Sorry it's possible I wasn't clear enough. I meant _just_ the username.

![image](https://user-images.githubusercontent.com/103725434/212616461-466468f7-a25e-4eff-ae65-33742e1a5dd5.png)

Otherwise there's a toggle in Developer tools > Flags for LDAP debug, if you want to troubleshoot.
